src/components/Communicator/CommunicatorDialog/CommunicatorDialogBoardItem.component.js

```diff
@@ -566,6 +566,10 @@ class CommunicatorDialogBoardItem extends React.Component {
                           <b>{intl.formatMessage(messages.boardInfoId)}:</b>{' '}
                           {board.id}
                         </Typography>
+                        <Typography variant="body1" gutterBottom>
+                          <b>{intl.formatMessage(messages.boardInfoUrl)}:</b>{' '}
+                          {'https://app.cboard.io/board/' + board.id}
```

Comment:
you should not use hard-coded value for a url. You should build the URL dynamically using the window js global object. See an example here: https://github.com/cboard-org/cboard/blob/2285a1cfb1b56e0bbbded2afa1562bb068bcca7b/src/constants.js#L10